site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-java-with-minio.md
         Slug: how-to-use-aws-sdk-for-java-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Java with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk-java is the official AWS SDK for the Java programming language. In this recipe we will learn how to use aws-sdk-java with MinIO server
+        SEO Keywords: AWS SDK for Java 
 
       - How to use Paperclip with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/how-to-use-paperclip-with-minio.md
         Slug: how-to-use-paperclip-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use Paperclip with MinIO Server - Cookbook/Recipe
+        SEO Description: Paperclip is intended as an easy file attachment library for ActiveRecord. In this recipe you will learn how to configure MinIO as an object storage backend for Paperclip.
+        SEO Keywords: Paperclip, ActiveRecord
 
       - How to use AWS SDK for .NET with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-dotnet-with-minio.md
         Slug: how-to-use-aws-sdk-for-net-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for .NET with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk-dotnet is the official AWS SDK for the .NET Framework. In this recipe we will learn how to use aws-sdk-dotnet with MinIO server.
+        SEO Keywords: AWS SDK for .NET
 
       - How to use Minio's server-side-encryption with aws-cli:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/how-to-use-minio-server-side-encryption-with-aws-cli.md
         Slug: how-to-use-minio-s-server-side-encryption-with-aws-cli
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use MinIO's server-side-encryption with aws-cli - Cookbook/Recipe
+        SEO Description: MinIO supports S3 server-side-encryption with customer provided keys (SSE-C), this describes the use of server-side encryption with AWS Command Line Interface (aws-cli)
+        SEO Keywords: aws-cli, server side encryption
 
       - How to monitor Minio using Prometheus:
         Link: https://github.com/minio/cookbook/blob/master/docs/how-to-monitor-minio-with-prometheus.md
         Slug: how-to-monitor-minio-using-prometheus
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to monitor MinIO using Prometheus - Cookbook/Recipe
+        SEO Description: This document explains how to setup Prometheus and configure it to scrape data from MinIO servers.
+        SEO Keywords: Prometheus
```

Comment:
Maybe we can have additional keywords to indicate that this documentation is about scraping data from MinIO servers and how to use Prometheus with MinIO